src/main/resources/com/google/api/codegen/py/method_sample.snip

```diff
@@ -174,26 +174,29 @@
 
 @snippet optionalArrayMethodSampleCodeBidiStreaming(apiMethod, initCode)
   requests = [request]
-  {@responseStreamingSampleCode(apiMethod, initCode)}
+  {@responseStreamingSampleCode(apiMethod, initCode, processElement())}
 @end
 
-@snippet pagedOptionalArrayMethodSampleCodeAll(apiMethod, initCode)
+@snippet pagedOptionalArrayMethodSampleCodeAll(apiMethod, initCode, processResp)
   @# Iterate over all results
-  {@responseStreamingSampleCode(apiMethod, initCode)}
+  {@responseStreamingSampleCode(apiMethod, initCode, processResp)}
 @end
 
-@snippet pagedOptionalArrayMethodSampleCodePaged(apiMethod, initCode)
+@snippet pagedOptionalArrayMethodSampleCodePaged(apiMethod, initCode, processResp)
```

Comment:
The following is giving me pause (which is why I didn't reply yesterday...):

We are re-using the template functions from this file in the temporary stubs in `standalone_sample.snip`. Those stubs will very likely get replaced by proper standalone sample-related template functions. In that case, changing the signature of these functions here that we're only using as temporary stubs there seems like adding unnecessary complexity. The alternative would be to use this as the opportunity to copy these functions into `standalone_sample.snip` and not depend on these, and those can have the new parameter.

I _think_ I'm leaning to accept what you have here because it's possible that we _may_ reuse some of these functions in the real standalone samples, not just the stubs. I'm just worried that if we don't, and wind up replacing the stubs after all, then the functions here will have crufty more-complicated signatures with the extra param that will no longer be needed.

Thoughts?